### DISCUSSION TITLE
DISPLAY TAG SEPEARATED

### DISCUSSION THREAD

**Discussion by subbareddyvs** [microsoft-search/pnp-modern-search#1545] [Q&A]
Hi,
We have added Enterprise keywords column to the library and want to display the kewords tagged in the search results.
They are displayed but without a separator for each tag, instead they are displayed as one word..  Can you please help how to split the keywords/tag.  Ex: below two different keywords
![image](https://user-images.githubusercontent.com/63510269/142776074-816ee86c-2889-42d8-995d-65cbed3f3dfd.png)

**Comment by wobba** [comment]
Not sure what version you are using. The list template in v4 has a space which was added for v4.4.
